apps/ethvm/src/states/actions.ts

```diff
@@ -3,22 +3,22 @@ import sEvents from '@app/configs/socketEvents.json'
 import { Block, Tx } from '@app/libs'
 import { BlockLayout, TxLayout } from '@app/typeLayouts'
 
-const socket_newBlock = ({ commit }, block: BlockLayout | BlockLayout[]) => {
+const socket_socketNewblock = function ({ commit }, block: BlockLayout | BlockLayout[])  {
   commit(sEvents.newBlock, block)
   this._vm.$eventHub.$emit(sEvents.newBlock, Array.isArray(block) ? new Block(block[0]) : new Block(block))
 }
 
-const socket_newTx = function({ commit }, tx: TxLayout | TxLayout[]) {
+const socket_socketNewtx = function ({ commit }, tx: TxLayout | TxLayout[]) {
   commit(sEvents.newTx, tx)
   this._vm.$eventHub.$emit(sEvents.newTx, Array.isArray(tx) ? new Tx(tx[0]) : new Tx(tx))
 }
 
-const socket_connect = ({}, tx: TxLayout) => {
+const socket_socketConnect = function ({}, tx: TxLayout) {
```

Comment:
Two related questions:

1) Is function needed? Or can we use shorter version with the arrow syntax?

2) Do we need to pass an object and a tx class as fn parameters? It's not being used, right?

Reply:
yea cz `this` wasnt working :(

Reply:
ouch! Then we need to add rules to avoid the linter complain!

Reply:
2. first parameter is `{ commit, state, dispatch }` if we need any of them we can use it, second parameter is the payload

Reply:
we can make $eventHub part of the state, that will let us go back to arrow functions

Reply:
Ok got it! Then decide which strategy is better in this particular case (both will work) 👍